tronbyt_server/manager.py

```diff
@@ -210,6 +210,7 @@ def create() -> ResponseReturnValue:
                 return redirect(url_for("manager.create"))
             if not img_url:
                 img_url = f"{server_root()}/{device_id}/next"
+                ws_url = ws_root() + f"/{device_id}/ws"
```

Comment:
What should we set `ws_url` to if  `img_url` is manually overridden? I don't handle that case at all. Probably the default still (effectively disallowing custom WebSocket URLs?)